### PR TITLE
#10 fix web RTC leak settings

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

File: Extension/lib/utils/webrtc.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Adguard Browser Extension.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* global chrome, Prefs */
+
+chrome.storage.local.get(null, function (items) {
```
Comment (ameshkov) [inline Extension/lib/utils/webrtc.js]: chrome?

File: Extension/lib/prefs.js [hunk 2]
```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
             }
             return browser;
         })(),
-        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0
+        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0,
```
Comment (ameshkov) [inline Extension/lib/prefs.js]: What's wrong with browser field?

File: Extension/lib/prefs.js [hunk 3]
```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
...
+        chromiumVersion: (function () {
```
Comment (ameshkov) [inline Extension/lib/prefs.js]: We could simply rename it to "version" as this field may be useful for other browsers

File: Extension/lib/prefs.js (hunk 2 above)
Comment (Mizzick) [inline Extension/lib/prefs.js]: what do you mean?

Comment (ameshkov) [inline Extension/lib/prefs.js]: I mean that we have Prefs.browser field, which already detects yandex browser. Why more fields?

Comment (Mizzick) [inline Extension/lib/prefs.js]: to have only one string "Yowser"?
I don't know, as you can see I didn't add this line, it was here before.

File: Extension/lib/utils/webrtc.js (hunk 1 above)
Comment (Mizzick) [inline Extension/lib/utils/webrtc.js]: Yes, why not?

File: Extension/lib/prefs.js (hunk 2 above)
Comment (ameshkov) [inline Extension/lib/prefs.js]: Hm, this is something specific to stealth mode i guess. We just should not migrate it to the main extension.

File: Extension/lib/utils/webrtc.js (hunk 1 above)
Comment (ameshkov) [inline Extension/lib/utils/webrtc.js]: Forgot that this is not a main extension:)

**Pull request merged**